### PR TITLE
feat: added documentation code lens

### PR DESCRIPTION
**Pull request by AdiGajbhiye**
## Overview

feat: added documentation code lens
Screenshot 2024-07-24 at 7 42 23 PM
Screenshot 2024-07-24 at 7 42 55 PM

### Problem

[
Describe the problem you are solving. Mention the ticket/issue if applicable.
](https://github.com/AltimateAI/vscode-dbt-power-user/issues/1292)
### Solution

Describe the implemented solution. Add external references if needed.

### Screenshot/Demo

A picture is worth a thousand words. Please highlight the changes if applicable.

### How to test

- Steps to be followed to verify the solution or code changes
- Mention if there is any settings configuration added/changed/deleted

## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

File: src/commands/index.ts [hunk 1]
```diff
@@ -654,6 +654,27 @@ export class VSCodeCommands implements Disposable {
...
+          const { event } = result;
+          if (!event) {
+            return;
+          }
+          const { nodeMetaMap } = event;
+          const model = nodeMetaMap.get(modelName);
+          if (!model?.path) {
+            return;
+          }
+          const doc = await workspace.openTextDocument(Uri.file(model.path));
```
Comment (mdesmet) [inline src/commands/index.ts:673]: Does this reopen the existing doc if it is already open in the editor area?

Comment (mdesmet) [reply src/commands/index.ts:673]: Is the path absolute or relative?

**Review by mdesmet — APPROVED** [review]
LGTM

File: src/commands/index.ts (hunk 1 above)
Comment (AdiGajbhiye) [reply src/commands/index.ts:673]: its absolute path. yeah. it opens existing tab for file

**Review by saravmajestic — APPROVED** [review]

**Pull request merged**